client/post-editor/media-modal/gallery-help.jsx

```diff
@@ -101,10 +107,33 @@ export default React.createClass( {
 	},
 
 	render() {
+		if ( this.props.isMediaModalGalleryInstructionsDismissed ) {
```

Comment:
Question: What will happen if preferences aren't stored in localStorage, but a pending request from the API will reveal that they've dismissed the preference before? Seems like it might display the help dialog briefly?